apps/services/auth/api/src/app/modules/delegations/delegations.controller.spec.ts

```diff
@@ -191,6 +212,34 @@ describe('DelegationsController', () => {
               )
             }
 
+            for (let i = 0; i < deceased; i++) {
+              const representedPerson: NameIdTuple = [
+                getFakeName(),
+                deceasedNationalIds[i],
+              ]
+              const relationship = getPersonalRepresentativeRelationship(
+                userKennitala,
+                representedPerson[1],
+              )
+
+              await prModel.create(relationship)
+              await prRightsModel.create(
+                getPersonalRepresentativeRights('valid1', relationship.id),
+              )
+              // Create a delegation for the deceased person
+              await delegationModel.create(
+                createDelegation({
+                  fromNationalId: deceasedNationalIds[i],
+                  toNationalId: user.nationalId,
+                  scopes: [defaultScopes.testUserHasAccess.name],
+                  today,
+                }),
+                {
+                  include: [{ model: DelegationScope, as: 'delegationScopes' }],
+                },
+              )
```

Comment:
I don't think there is a delegation in personal representative relationships.

Reply:
this has been removed